events/wc_events/wc_exodar_crash_events.txt

```diff
@@ -305,33 +304,100 @@ exodar.1009 = {
 	lower_center_portrait = { character = scope:ishanah }
 	lower_right_portrait = { character = scope:admetius }
 
+	immediate = {
+		if = {
+			limit = {
+				NOR = {
+					in_diplomatic_range = scope:velen
+					in_diplomatic_range = scope:maraad
+					in_diplomatic_range = scope:proenitus
+					in_diplomatic_range = scope:admetius
+					in_diplomatic_range = scope:ishanah
+				}
+			}
+			add_character_flag = viewer
+		}
+	}
+
 	option = {
-		trigger = { scope:maraad = { is_ai = yes } }
-		name = exodar.1004.a
+		trigger = { 
+			AND = {
+				NOT = { has_character_flag = viewer }
+				scope:maraad = { is_ai = yes }
+			}
+		}
+		name = exodar.1009.a
 		set_player_character = scope:maraad
 	}
 
 	option = {
-		trigger = { scope:proenitus = { is_ai = yes } }
-		name = exodar.1004.b
+		trigger = { 
+			AND = {
+				NOT = { has_character_flag = viewer }
+				scope:proenitus = { is_ai = yes }
+			}
+		}
+		name = exodar.1009.b
 		set_player_character = scope:proenitus
 	}
 
 	option = {
-		trigger = { scope:admetius = { is_ai = yes } }
-		name = exodar.1004.c
+		trigger = { 
+			AND = {
+				NOT = { has_character_flag = viewer }
+				scope:admetius = { is_ai = yes }
+			}
+		}
+		name = exodar.1009.c
 		set_player_character = scope:admetius
 	}
 
 	option = {
-		trigger = { scope:ishanah = { is_ai = yes } }
-		name = exodar.1004.d
+		trigger = { 
+			AND = {
+				NOT = { has_character_flag = viewer }
+				scope:ishanah = { is_ai = yes }
+			}
+		}
+		name = exodar.1009.d
 		set_player_character = scope:ishanah
 	}
 
 	option = {
-		name = exodar.1004.e
-		#nothing happens
-		custom_tooltip = exodar.1004.e.tooltip
+		trigger = { NOT = { has_character_flag = viewer } }
+		name = exodar.1009.e
+		custom_tooltip = exodar.1009.e.tooltip
+	}
+	
+	option = {
+		trigger = { has_character_flag = viewer }
+		name = {
+			text = {
+				first_valid = {
+					triggered_desc = {
+						trigger = { religion = { is_in_family = rf_disorder } }
+						desc = exodar.1001.a
+					}
+					triggered_desc = {	
+						trigger = {  religion = { is_in_family = rf_light } }
+						desc = exodar.1001.b
+					}
+					#One of the elves that fought Archimonde?
+					triggered_desc = {
+						trigger = {
+							OR = {
+								culture = { has_cultural_pillar = heritage_darnassian }
+								culture = { has_cultural_pillar = heritage_highborne }
+								culture = { has_cultural_pillar = heritage_draconic }
+								culture = { has_cultural_pillar = heritage_cenaric }
+							}
+						}
+						desc = exodar.1001.c
+					}
+					desc = exodar.1001.d
+				}
+			}
+		}
+		remove_character_flag = viewer
```

Comment:
whoever reviews this, please pay special attention to the changes in this file. I'm not sure where they came from as they just kinda popped up on my branch when i tried to make changes to the localization paths for exodar.1009. it looks like they might have been removed in master and i was working off an old branch? i just want confirmation to put them back before i delete anything.

Reply:
Add this back yes, this removes the flag after the event is over.